### PR TITLE
RoundRobin.md

### PR DESCRIPTION
**Pull request by Pule08**

### CODE REVIEW

**Review by MaxHillebrand — APPROVED** [review]

**Review by nopara73 — CHANGES_REQUESTED** [review]
Very good. Make sure to update https://github.com/zkSNACKs/zIPs#readme as well

**Review by MaxHillebrand — APPROVED** [review]

**Review by molnard — APPROVED** [review]

**Comment by molnard** [comment]
@Pule08 do you know how to update https://github.com/zkSNACKs/zIPs#readme in this PR?

File: RoundRobin.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+```
+zIP: 7
+Layer: Development
+Title: Round Robin
+Author: molnard, pule08
+Status: Proposed
+Type: Active
+Created: 2023-03-30
+License: lol
```
[the remark this answers is not in the document]
Comment (Pule08) [reply RoundRobin.md:11]: I copied it from the relevance zIP.

[the remark this answers is not in the document]
Comment (nopara73) [reply RoundRobin.md:11]: "LOL" is an acronym that stands for "laugh out loud." It is commonly used in text messages, social media, and online conversations to indicate that the sender finds something funny or amusing. The phrase is typically used to express humor, amusement, or lightheartedness.

**Comment by nopara73** [comment]
Sister PR: https://github.com/zkSNACKs/zIPs/pull/93

File: RoundRobin.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pule08) [reply RoundRobin.md:11]: That was my first thought :D

**Review by molnard — APPROVED** [review]
🥇

**Pull request merged**